app/controllers/api/framework_responses_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Api
+  class FrameworkResponsesController < ApiController
+    PPERMITTED_PARAMS = [
+      :type,
+      attributes: [:value, { value: %i[option details] }, { value: [] }],
```

Comment:
Interesting syntax! Might be worth a comment here to explain as it's not immediately clear what shape the params have. Also I can't see any mention of `details` in the specs - there's one case for passing `options`.